content/docs/dvclive/api-reference/init.md

```diff
@@ -0,0 +1,61 @@
+# dvclive.init()
+
+Initializes a DVCLive logger.
+
+```py
+def init(
+    path: str = None,
+    resume: bool = False,
+    step: int = 0,
+    summary: bool = True,
+    html: bool = True,
+)
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.init()
+```
+
+## Description
+
+It's usage is optional and focused on configuring the behavior of subsequent
+calls to [`dvclive.log`](/doc/dvclive/api-reference/log) and
```

Comment:
future improvement: we should create a ticket or improve auto-linker, to keep text clean from these internal links like we do in DVC API reference

Reply:
https://github.com/iterative/dvclive/issues/114